include/dlpack/dlpack.h

```diff
@@ -114,6 +114,26 @@ typedef struct {
   /*! \brief The offset in bytes to the beginning pointer to data */
   uint64_t byte_offset;
 } DLTensor;
+
+/*!
+ * \brief C Tensor object, manage memory of DLTensor. This data structure is
+ *  intended to faciliate the borrowing of DLTensor by another framework. It is
+ *  not meant to transfer the tensor. When the borrowing framework doesn't need
+ *  the tensor, it should call the deleter to handle reference counting.
+ */
+struct DLManagedTensor {
```

Comment:
We still want to keep typedef style. Otherwise, the reference to DLManagedTensor always has to be ```struct DLManagedTensor``` in C

Reply:
i.e. there always has to be a struct prefix

Reply:
consider add ```typedef struct DLManagedTensor DLManagedTensor;```